spec/apis/single-spa-events-api.spec.js

```diff
@@ -566,7 +566,9 @@ describe(`events api :`, () => {
         });
     });
 
-    it(`allows you to cancel a pushState navigation event in the before-routing-event handler`, (done) => {
+    it(`allows you to cancel a pushState navigation event in the before-routing-event handler`, async () => {
```

Comment:
This test was brittle and would have different results if you ran it individually versus together with other tests. I have fixed it by ensuring that no previous url changes from other tests impact its behavior.